examples/channels-3u-1d.rs

```diff
@@ -0,0 +1,91 @@
+#![deny(unsafe_code)]
+#![no_std]
+#![no_main]
+
+use cortex_m_rt::entry;
+use embedded_hal::blocking::delay::DelayMs;
+use nucleo_f446re::{led::LedDigital, Nucleo};
+use panic_probe as _;
+
+
+use core::fmt::Write;
+use rtt_target::{rtt_init, ChannelMode::BlockIfFull};
+
+#[entry]
+fn main() -> ! {
+    let mut nucleo = Nucleo::<LedDigital>::init().unwrap();
+
+    let channels = rtt_init! {
+        up: {
+            0: {
+                size: 512,
+                mode: BlockIfFull,
+                name: "Up zero"
+            }
+            1: {
+                size: 128,
+                name: "Up one"
+            }
+            2: {
+                size: 128,
+                name: "Up two"
+            }
+        }
+        down: {
+            0: {
+                size: 512,
+                mode: BlockIfFull,
+                name: "Down zero"
+            }
+        }
+    };
+
+    let mut output2 = channels.up.1;
+    writeln!(
+        output2,
+        "Hi! I will turn anything you type on channel 0 into upper case."
```

Comment:
I ran this example but couldn't get it to work like I thought. On channel 0 I am able to get an uppercase'd echo and periodic logs on channel 2, but channel 1 (which has this message) doesn't do anything. I probably need to read up more on RTT to get what is going on here.

It does feel similar to the comment I left on the other example, that I'm not sure this adds much to this nucleo crate (rather an example of RTT usage). But, it is a more involved example so I'm more inclined to include it.

Reply:
Removing for now